<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tripReceipt</class>
 <widget class="QDialog" name="tripReceipt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="tripTotalDisplay_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalDistanceDisplay_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>total Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="schoolTotalDisplay_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>school total</string>
   </property>
  </widget>
  <widget class="QComboBox" name="school_comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tripOrder_tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>271</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="souvenirsBySchool_tableView">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>301</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Reciept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
